chrome.manifest

```diff
@@ -1 +1,2 @@
+content usearch chrome/
```

Comment:
Is it possible to use better nomenclature for this? `universalsearch` would be so much clearer.